### PR TITLE
Add react query support

### PR DESCRIPTION
**Pull request by Nepomuceno**
### What does this PR do?

Add support for react query to get the data from the api.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

File: .github/workflows/main_iqengineweb.yml [hunk 1]
```diff
@@ -63,64 +64,3 @@ jobs:
           slot-name: 'staging'
           publish-profile: ${{ secrets.AzureAppService_PublishProfile_Staging }}
           package: .
-  
-  # selenium-tests:
```
Comment (Jaydee123) [inline .github/workflows/main_iqengineweb.yml:67]: How come we have removed this?

Comment (Jaydee123) [reply .github/workflows/main_iqengineweb.yml:67]: Didn't notice that it was commented out, apologies!

**Comment by Jaydee123** [comment]
I love it, it's perfect <3

**Review by Nepomuceno — COMMENTED** [review]
LGTM

**Pull request merged**